proposals/dictionary-expressions.md

```diff
@@ -220,33 +281,48 @@ The new rules above represent a breaking change: For types that are a valid conv
 > For the create method:
 >   - The method must have a single parameter of type System.ReadOnlySpan<E>, passed by value, and there is an identity conversion from E to the iteration type of the collection type.
 >    - ```diff
->      + Or, the method have a single parameter of
+>      + Or, the method has a single parameter of
 >      + `IEnumerable<KeyValuePair<TKey, TValue>>` and iteration type
 >      + of the collection type is the same `KeyValuePair<,>` type. 
+>   - The method has two parameters, where the first is an `IEqualityComparer<TKey>` and the second follows the rules of the 'single parameter' rule above.  This method will be called if the collection expression includes an initial `comparer: expression`.
 > ```
```

Comment:
Let's use **bold** for the changes rather than `diff`:

> A create method is indicated with a [CollectionBuilder(...)] attribute on the collection type. The attribute specifies the builder type and method name of a method to be invoked to construct an instance of the collection type.
> 
> **A create method will commonly use the name `CreateRange` in the dictionary domain.**
>
> For the create method:
>   - The method must have a single parameter of type System.ReadOnlySpan<E>, passed by value, and there is an identity conversion from E to the iteration type of the collection type, **or the method has a single parameter of `IEnumerable<KeyValuePair<TKey, TValue>>` and iteration type of the collection type is the same `KeyValuePair<,>` type.**
>   - The method has two parameters, where the first is an `IEqualityComparer<TKey>` and the second follows the rules of the 'single parameter' rule above.  This method will be called if the collection expression includes an initial `comparer: expression`.